actions/pdp-renderer/render.js

```diff
@@ -20,6 +21,20 @@ function toTemplateProductData(baseProduct) {
   templateProductData.primaryImage = primaryImage;
   templateProductData.metaTitle = baseProduct.metaTitle || baseProduct.name || 'Product Details';
 
+  // For each field which can contain HTML, validate it. If it is invalid, strip the HTML and log a warning.
+  const htmlFieldValidations = {
+    metaDescription: validateHtml(templateProductData.metaDescription),
+    shortDescription: validateHtml(templateProductData.shortDescription),
+    description: validateHtml(templateProductData.description)
+  }
+
+  Object.entries(htmlFieldValidations).forEach(([field, validation]) => {
+    if (!validation.valid) {
+      templateProductData[field] = '';
+      context.logger.warn(`HTML validation failed for "${field}" field: ${validation.reason}`);
```

Comment:
Is this log sufficient to inform the user? Hopefully users do not have to experience what I did - tracing backwards from malformed UX through EDS CDN, Azure, and finally realizing it is the data itself that is the problem.

@duynguyen let me know if there's a better way to alert users to bad data.